awx/ui/client/src/templates/job_templates/multi-credential/multi-credential-modal.directive.js

```diff
@@ -111,7 +111,7 @@ function multiCredentialModalController(GetBasePath, qs, MultiCredentialService)
 
         scope.credentialTypes.forEach((credentialType => {
             if(credentialType.kind
-                .match(/^(machine|cloud|net|ssh|vault)$/)) {
+                .match(/^(machine|cloud|net|ssh|vault|kubernetes)$/)) {
```

Comment:
I'd check myself but I'm running into an issue where my download speed drops to 40 kb/s while building the image locally and don't want to tie up my computer for 45 minutes:

![Screenshot from 2020-07-16 08-24-17](https://user-images.githubusercontent.com/9753817/87670496-cfb68800-c73d-11ea-8817-47f8e4471e71.png)

... so I'll just ask: Is the credential type displayed on the job launch credential prompt as well? If not we'll need to update some more code & you'd want to start here: https://github.com/ansible/awx/blob/devel/awx/ui/client/src/templates/prompt/steps/credential/prompt-credential.controller.js


We'll also want to check the form, job details, and prompt on the new UI.

Reply:
ALRIGHT!!, I made it, finally :confetti_ball: